test/NuGet.Core.Tests/NuGet.PackageManagement.Test/BuildIntegratedTests.cs

```diff
@@ -1848,14 +1851,2073 @@ public async Task TestPacMan_BuildIntegrated_PreviewUpdatesAsync_WithStrictVersi
             }
         }
 
-        private SourceRepositoryProvider CreateSource(List<SourcePackageDependencyInfo> packages)
+        [Fact]
+        public async Task TestPacMan_PreviewBuildIntegratedProjectsActionsAsync_InstallPackageForAllProjects()
+        {
+            // Arrange
+            var sources = new List<PackageSource>();          
+            var projectDirectories = new List<TestDirectory>();
+
+            try
+            {
+                using (var settingsDirectory = TestDirectory.Create())
```

Comment:
You are defining this settings directory, but you are not using it.

Reply:
Removed this when porting those unit test to NetCorePackageReferenceProjectTests.cs